app/models/dao_revocation.rb

```diff
@@ -0,0 +1,15 @@
+class DaoRevocation < ApplicationRecord
+  belongs_to :project, dependent: :destroy
```

Comment:
This feels iffy to me, destroying the revocation destroys the parent Project? I would expect a Conversion::Project to declare a `dao_revocation` association with the destroy on it but not the other way around - am I missing some of the context or design?

Do you think we need to think about making it fast to scope Projects that have a DAO Revocation, perhaps by reversing this association, it feels wrong that a 'project belongs to a DAO revocation' but then we can do something like:

`Conversion::Project.where.not(dao_revocation_id: nil)`

to get all DAO project?

Reply:
No putting the destroy on the wrong bit was my fault!